library/Vanilla/Models/FsThemeProvider.php

```diff
@@ -133,6 +133,23 @@ public function getThemeAddon($themeKey): AddonInterface {
         return $theme;
     }
 
+    /**
+     * Verify if a theme exists by it's theme identifier.
+     *
+     * @param string|int $themeKey
+     * @return bool
+     */
+    public function themeExists($themeKey): bool {
+        $themeExists = true;
+        $theme = $this->addonManager->lookupTheme($themeKey);
+
+        if (!($theme instanceof AddonInterface) || is_null($theme)) {
+            $themeExists = false;
+        }
```

Comment:
Can't we default to `$themeExists = false;` until we check it and have any proof?
```suggestion
        $themeExists = false;
        $theme = $this->addonManager->lookupTheme($themeKey);
        if ($theme instanceof AddonInterface) {
            $themeExists = true;
        }
```